bftengine/src/preprocessor/messages/PreProcessBatchRequestMsg.hpp

```diff
@@ -65,6 +65,7 @@ class PreProcessBatchRequestMsg : public MessageBase {
   Header* msgBody() const { return ((Header*)msgBody_); }
 
  private:
+  inline bool checkElements() const;
```

Comment:
I think you should remove the inline keyword, as the function is not defined in the header.